fakemails/rsync.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+account=$1
+uid=$2
+gid=$3
+fakedir=$4
+
+echo "Copying fake emails to the $account email account."
+rsync -a --no-perms --chown $uid:$gid $fakedir/* "/var/mail/$account/Maildir/cur/"
```

Comment:
The rsync command should include the `--delete` flag to ensure that the destination directory reflects the source directory exactly, removing any files that no longer exist in the source.
```suggestion
rsync -a --delete --no-perms --chown $uid:$gid $fakedir/* "/var/mail/$account/Maildir/cur/"
```